terraform/modules/prom-ec2/prometheus/variables.tf

```diff
@@ -7,7 +7,7 @@ variable "device_mount_path" {
 
 variable "data_volume_size" {
   description = "The size of the volume that will contain the prometheus data"
-  default     = 50
+  default     = 250
```

Comment:
can we only change this in prod, not in staging too?

Reply:
I want to test the resize logic works on way through staging ... but can lower it in PR after

Reply:
just set the variable in terraform/projects/prom-ec2/paas-production/main.tf